impl/client/src/main/javascript/web/vizapi/ccc/sample/sampleVisualization.js

```diff
@@ -121,7 +124,7 @@ require(["common-ui/vizapi/VizController"], function(){
 
     }
 
-    this.numSpan.innerHTML = value == null ? "" : value;
+    this.numSpan.innerHTML = value == null ? "" : DOMPurify.sanitize(value);
```

Comment:
This one should be changed to `innerText` given value is a null or a number and cannot possibly contain any HTML.

Reply:
Makes sense and change done.